### PR TITLE
Development

### PR DESCRIPTION
**Pull request by salac-kolosek**

### CODE REVIEW

File: test/models/user_test.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
```
Comment (sniffler) [inline test/models/user_test.rb]: Missing magic comment # frozen_string_literal: true.

File: spec/models/user_spec.rb [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+require 'rails_helper'
```
Comment (sniffler) [inline spec/models/user_spec.rb]: Add an empty line after magic comments.

File: spec/features/users_spec.rb [hunk 3]
```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe 'Users' do
```
Comment (sniffler) [inline spec/features/users_spec.rb]: Block has too many lines. [30/25]

File: db/schema.rb [hunk 4]
```diff
@@ -10,9 +10,26 @@
...
+    t.datetime "remember_created_at"
+    t.integer  "sign_in_count",          default: 0,  null: false
+    t.datetime "current_sign_in_at"
+    t.datetime "last_sign_in_at"
+    t.inet     "current_sign_in_ip"
+    t.inet     "last_sign_in_ip"
+    t.datetime "created_at",                          null: false
+    t.datetime "updated_at",                          null: false
+    t.index ["email"], name: "index_users_on_email", unique: true, using: :btree
+    t.index ["reset_password_token"], name: "index_users_on_reset_password_token", unique: true, using: :btree
```
Comment (sniffler) [inline db/schema.rb]: Prefer single-quoted strings when you don't need string interpolation or special symbols.<br>Line is too long. [110/80]

File: db/schema.rb [hunk 5]
```diff
@@ -10,9 +10,26 @@
...
+    t.datetime "reset_password_sent_at"
+    t.datetime "remember_created_at"
+    t.integer  "sign_in_count",          default: 0,  null: false
+    t.datetime "current_sign_in_at"
+    t.datetime "last_sign_in_at"
+    t.inet     "current_sign_in_ip"
+    t.inet     "last_sign_in_ip"
+    t.datetime "created_at",                          null: false
+    t.datetime "updated_at",                          null: false
+    t.index ["email"], name: "index_users_on_email", unique: true, using: :btree
```
Comment (sniffler) [inline db/schema.rb]: Prefer single-quoted strings when you don't need string interpolation or special symbols.

File: db/schema.rb [hunk 6]
```diff
@@ -10,9 +10,26 @@
...
+    t.string   "reset_password_token"
+    t.datetime "reset_password_sent_at"
+    t.datetime "remember_created_at"
+    t.integer  "sign_in_count",          default: 0,  null: false
+    t.datetime "current_sign_in_at"
+    t.datetime "last_sign_in_at"
+    t.inet     "current_sign_in_ip"
+    t.inet     "last_sign_in_ip"
+    t.datetime "created_at",                          null: false
+    t.datetime "updated_at",                          null: false
```
Comment (sniffler) [inline db/schema.rb]: Prefer single-quoted strings when you don't need string interpolation or special symbols.

File: db/schema.rb [hunk 7]
```diff
@@ -10,9 +10,26 @@
...
+    t.string   "full_name",                           null: false
+    t.string   "reset_password_token"
+    t.datetime "reset_password_sent_at"
+    t.datetime "remember_created_at"
+    t.integer  "sign_in_count",          default: 0,  null: false
+    t.datetime "current_sign_in_at"
+    t.datetime "last_sign_in_at"
+    t.inet     "current_sign_in_ip"
+    t.inet     "last_sign_in_ip"
+    t.datetime "created_at",                          null: false
```
Comment (sniffler) [inline db/schema.rb]: Prefer single-quoted strings when you don't need string interpolation or special symbols.

File: db/schema.rb [hunk 8]
```diff
@@ -10,9 +10,26 @@
...
+    t.string   "encrypted_password",     default: "", null: false
+    t.string   "full_name",                           null: false
+    t.string   "reset_password_token"
+    t.datetime "reset_password_sent_at"
+    t.datetime "remember_created_at"
+    t.integer  "sign_in_count",          default: 0,  null: false
+    t.datetime "current_sign_in_at"
+    t.datetime "last_sign_in_at"
+    t.inet     "current_sign_in_ip"
+    t.inet     "last_sign_in_ip"
```
Comment (sniffler) [inline db/schema.rb]: Prefer single-quoted strings when you don't need string interpolation or special symbols.

File: db/schema.rb [hunk 9]
```diff
@@ -10,9 +10,26 @@
...
+    t.string   "email",                  default: "", null: false
+    t.string   "encrypted_password",     default: "", null: false
+    t.string   "full_name",                           null: false
+    t.string   "reset_password_token"
+    t.datetime "reset_password_sent_at"
+    t.datetime "remember_created_at"
+    t.integer  "sign_in_count",          default: 0,  null: false
+    t.datetime "current_sign_in_at"
+    t.datetime "last_sign_in_at"
+    t.inet     "current_sign_in_ip"
```
Comment (sniffler) [inline db/schema.rb]: Prefer single-quoted strings when you don't need string interpolation or special symbols.

File: db/schema.rb [hunk 10]
```diff
@@ -10,9 +10,26 @@
...
+  create_table "users", force: :cascade do |t|
+    t.string   "email",                  default: "", null: false
+    t.string   "encrypted_password",     default: "", null: false
+    t.string   "full_name",                           null: false
+    t.string   "reset_password_token"
+    t.datetime "reset_password_sent_at"
+    t.datetime "remember_created_at"
+    t.integer  "sign_in_count",          default: 0,  null: false
+    t.datetime "current_sign_in_at"
+    t.datetime "last_sign_in_at"
```
Comment (sniffler) [inline db/schema.rb]: Prefer single-quoted strings when you don't need string interpolation or special symbols.

File: app/policies/application_policy.rb [hunk 11]
```diff
@@ -0,0 +1,53 @@
...
+  def update?
+    false
+  end
+
+  def edit?
+    update?
+  end
+
+  def destroy?
+    false
```
Comment (stefan-kolosek) [inline app/policies/application_policy.rb]: Are these methods returning false necessary?

File: app/views/layouts/application.html.erb [hunk 12]
```diff
@@ -9,11 +9,12 @@
...
     <script src="https://use.fontawesome.com/90b92514d7.js"></script>
+    <%= render "layouts/application/flash_js" %>
   </head>
 
   <body>
+    <%= render "layouts/application/flash_messages" %>
     <%= yield %>
-    <script src="https://code.jquery.com/jquery-3.2.1.slim.min.js" integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN" crossorigin="anonymous"></script>
     <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js" integrity="sha384-ApNbgh9B+Y1QKtv3Rn7W3mgPxhU9K/ScQsAP7hUibX39j7fakFPskvXusvfa0b4Q" crossorigin="anonymous"></script>
     <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta.3/js/bootstrap.min.js" integrity="sha384-a5N7Y/aK3qNeh15eJKGWxsqtnX/wWdSZSKp+81YjTmS15nvnvxKHuzaWwXHDli+4" crossorigin="anonymous"></script>
```
Comment (stefan-kolosek) [inline app/views/layouts/application.html.erb]: Move these to head please.

File: test/models/user_test.rb [hunk 13]
```diff
@@ -0,0 +1,7 @@
...
+
+class UserTest < ActiveSupport::TestCase
```
Comment (stefan-kolosek) [inline test/models/user_test.rb]: Try not to push empty tests, either add basic test cases or remove them.

**Pull request merged**